frontend/src/components/common/authorized-wrapper/authorized-wrapper.tsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
-import { RootState } from 'common/types';
 import { useSelector } from 'react-redux';
+import { RootState } from 'common/types';
+import { PermissionName } from 'common/enums';
 import { Header } from 'components/common';
+import CovidButton from 'components/covid-button/covid-button';
 import { IUserWithPermissions } from 'common/interfaces';
+import { checkHasPermission } from '../authorized-route/helpers';
 
 const AuthorizedWrapper: React.FC = ({ children }) => {
   const { user } = useSelector(({ auth }: RootState) => ({
     user: auth.user,
   }));
 
+  const hasMapPermission = checkHasPermission([PermissionName.MAP_MANIPULATION], user?.permissions ?? []);
```

Comment:
we use this helper in several places, so move it to the `frontend/helpers` and use it from there